src/skin.cpp

```diff
@@ -545,6 +546,31 @@ Polygons SkinInfillAreaComputation::generateNoAirAbove(SliceLayerPart& part)
     return no_air_above;
 }
 
+/*
+ * This function is executed in a parallel region based on layer_nr.
+ * When modifying make sure any changes does not introduce data races.
+ *
+ * this function may only read the skin and infill from the *current* layer.
+ */
+    Polygons SkinInfillAreaComputation::generateNoAirBelow(SliceLayerPart& part, size_t flooring_layer_count)
+    {
+        const size_t wall_idx = std::min(size_t(2), mesh.settings.get<size_t>("wall_line_count"));
```

Comment:
Use constexpr and add early out
```suggestion
        if (layer_nr <= flooring_layer_count)
        {
            return {};
        }
        constexpr size_t min_wall_line_count = 2;
        const size_t wall_idx = std::min(min_wall_line_count, mesh.settings.get<size_t>("wall_line_count"));
```